src/components/MemberDetails.tsx

```diff
@@ -63,10 +75,10 @@ export function MemberDetails({ member }: MemberDetailsProps) {
             </Serif>
             <span>
               {orgs.map((org) => (
-                <Fragment key={`org-${normalizeParam(org)}`}>
-                  <RouterLink href={`/org/${normalizeParam(org)}`} passHref>
+                <Fragment key={`org-${org.slug}`}>
```

Comment:
Frontloaded all the slug generation. Actually use all the slugs as ids. Makes things much easier.